src/main/java/rx/schedulers/EventLoopsScheduler.java

```diff
@@ -27,7 +27,7 @@
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
 
-/* package */class EventLoopsScheduler extends Scheduler {
+public class EventLoopsScheduler extends Scheduler {
```

Comment:
This makes this part of the public API. Why? 

For example: http://reactivex.io/RxJava/javadoc/rx/schedulers/package-frame.html